src/frontends/pytorch/src/op/log2.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset8.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_log2(NodeContext& context) {
+    auto x = context.get_input(0);
+    auto log2 = context.mark_node(opset8::Constant::create(element::f32, Shape{}, {0.693147180559945309}));
+    auto log = context.mark_node(std::make_shared<opset8::Log>(x));
+    auto log_f = context.mark_node(std::make_shared<opset8::Convert>(log, element::f32));
```

Comment:
We cast to f32 here, but do we need to convert back to initial type? Lets say input is f16 I think it make sense to convert to f16, but for integer types it is a question. How torch behaves in that case?

Reply:
it returns float32 for ints. probably we need some helpers for types checking like is_float or is_int

Reply:
added such logic